src/SignalR/clients/ts/jest.config.js

```diff
@@ -4,7 +4,7 @@
 module.exports = {
     globals: {
         "ts-jest": {
-            "tsConfig": "./tsconfig.jest.json",
```

Comment:
Probably need to revert this and the same change in FunctionalTests, it is only relevant when updating ts-jest to v26 or v27